src/main/markdown/changelog.md

```diff
@@ -1,5 +1,12 @@
 # Lagom Change Log
 
+## More recent releases
+
+Lagom team is now using Github Releases as the place to list changes in each release. We consider it is better since you can more easily consume all the issues, code changes, and other information there. Lagom releases on Github:
+
+<https://github.com/lagom/lagom/releases>
+
+Because of that, we are not keeping this page anymore, but we aren't removing it either for historical reasons.
```

Comment:
```suggestion
Because of that, we are not updating this page anymore, but we aren't removing it either for historical reasons.
```

Reply:
```suggestion
Because of that, we are not keeping this page up to date anymore, but we aren't removing it either for historical reasons.
```

Reply:
Oh, I just realized I can make multiple suggestions and let the PR owner choose the best fit. 👆🏼

Reply:
IANAL(inguist) but `we are not keeping the page anymore` sounds to me like we're getting rid of it eventually, when we actually intend to keep it forever.

Reply:
```suggestion
Because of that, we are not keeping this page updated anymore, but we aren't removing it either for historical reasons.
```